src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import java.util.Scanner;
+
 public class Duke {
+    public Duke() {
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String botName = "Hirofumi";
+        System.out.println("************************************************************");
+        System.out.println(" Hello! I'm " + botName);
+        System.out.println(" What can I do for you?");
+        System.out.println("************************************************************");
+
+        Scanner in = new Scanner(System.in);
+        String userInput = in.nextLine();
+        boolean isRunning = true;
+
+        int MAX_NUMBERED_LIST_LENGTH = 100;
+        Task[] numberedList = new Task[MAX_NUMBERED_LIST_LENGTH];
+        int listCount = 0;
+
```

Comment:
Perhaps it will help to move the different cases into methods so that it will be more readable